content/guides/core/hoon-school/B-syntax.md

```diff
@@ -543,7 +543,7 @@ Carefully map how the runes in that statement relate to each other, and notice h
 
 ##  Exercise:  “Absolute” Value (Around Ten)
 
```

Comment:
It’s with ^s.

Reply:
that one's for superscript but i need subscript

its ok tho, while `₁₀` looks a little janky in monospace font, it looks fine in normal font ₁₀